provider/core_internet_gateway_resource_test.go

```diff
@@ -52,6 +52,7 @@ func (s *ResourceCoreInternetGatewayTestSuite) TestAccResourceCoreInternetGatewa
 					resource.TestCheckResourceAttrSet(s.ResourceName, "id"),
 					resource.TestCheckResourceAttrSet(s.ResourceName, "time_created"),
 					resource.TestCheckResourceAttrSet(s.ResourceName, "display_name"),
+					resource.TestCheckResourceAttrSet(s.ResourceName, "vcn_id"),
```

Comment:
If you add this to the test but don't change provider/core_internet_gateway_resource.go, does this test still pass? I'm betting it does. (Either way you should add this - but generally I think we need to figure out a better testing strategy for import.)